islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -69,7 +65,7 @@ function islandora_paged_content_pdf_batch_drush_command() {
       ),
       'namespace' => array(
         'description' => dt('The namespace for objects created by this command.  Defaults to namespace set in Fedora config.'),
-        'required' => FALSE,
+        'required' => TRUE,
```

Comment:
It must have been a local problem.  We may have not specified a default namespace, since we did multisite (or maybe for some other reason).  The drush command does work for me if I specify a namespace, and reasonable to assume it works for others even without specifying one.  

No need to make "--namespace" required for other institutions, if ours is the only one affected.  We probably just had something configured different.  

Also, I'd added text in one error message, which related to a specific problem I was having with languages not being initialized.  It may also be irrelevant and not useful to other institutions.

But the lines relating the "target" and "scan_target", i believe, are helpful to everyone.  They're the only important lines in this commit, in my opinion.